scripts/install/readarr.sh

```diff
@@ -0,0 +1,105 @@
+
+    readarrConfDir="/home/$readarrOwner/.config/readarr"
+    mkdir -p "$readarrConfDir"
+    chown -R "$readarrOwner":"$readarrOwner" /home/$readarrOwner/.config
+
+    echo_progress_start "Downloading source files"
+    if ! curl "https://readarr.servarr.com/v1/update/nightly/updatefile?os=linux&runtime=netcore&arch=x64" -L -o /tmp/readarr.tar.gz >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Source downloaded"
+
+    echo_progress_start "Extracting archive"
+    tar -xvf /tmp/readarr.tar.gz -C /opt >> "$log" 2>&1
+    echo_progress_done "Archive extracted"
+
+    touch /install/.readarr.lock
+
+    echo_progress_start "Installing Systemd service"
+    cat > /etc/systemd/system/readarr.service << EOF
+[Unit]
+Description=readarr Daemon
+After=syslog.target network.target
+
+[Service]
+# Change the user and group variables here.
+User=${readarrOwner}
+Group=${readarrOwner}
+
+Type=simple
+
+# Change the path to readarr or mono here if it is in a different location for you.
+ExecStart=/opt/readarr/readarr -nobrowser
+TimeoutStopSec=20
+KillMode=process
+Restart=on-failure
+
+# These lines optionally isolate (sandbox) readarr from the rest of the system.
+# Make sure to add any paths it might use to the list below (space-separated).
+#ReadWritePaths=/opt/readarr /path/to/movies/folder
+#ProtectSystem=strict
+#PrivateDevices=true
+#ProtectHome=true
+
+[Install]
+WantedBy=multi-user.target
+EOF
+    chown -R "$readarrOwner":"$readarrOwner" /opt/readarr
+    systemctl -q daemon-reload
+    systemctl enable --now -q readarr
+    sleep 1
+    echo_progress_done "readarr service installed and enabled"
+
+    if [[ -f $readarrConfDir/update_required ]]; then
+        echo_progress_start "readarr is installing an internal upgrade..."
+        # echo "You can track the update by running \`systemctl status readarr\`0. in another shell."
+        # echo "In case of errors, please press CTRL+C and run \`box remove readarr\` in this shell and check in with us in the Discord"
+        while [[ -f $readarrConfDir/update_required ]]; do
+            sleep 1
+            echo_log_only "Upgrade file is still here"
+        done
+        echo_progress_done "Upgrade finished"
+    fi
+
+}
+
+_nginx_readarr() {
+    if [[ -f /install/.nginx.lock ]]; then
+        echo_progress_start "Installing nginx configuration"
+        #TODO what is this sleep here for? See if this can be fixed by doing a check for whatever it needs to
+        sleep 10
+        bash /usr/local/bin/swizzin/nginx/readarr.sh
+        systemctl -q reload nginx
+        echo_progress_done "Nginx configured"
+    else
+        echo_info "readarr will be available on port 8787. Secure your installation manually through the web interface."
+    fi
+}
+
+if [[ -z $readarrOwner ]]; then
+    readarrOwner=$(_get_master_username)
+fi
+
+_install_readarr
+_nginx_readarr
+
+if [[ -f /install/.ombi.lock ]]; then
```

Comment:
These checks were present for the v3 installers as APIs changed, I think these can be removed for a new app